### PR TITLE
[close #718] Implement TiFlash alive rpc

### LINKED ISSUE #718: Probe the status of TiFlash
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

StoreClient can not get all the status of TiFlash. 

 For example, if TiFlash is `disconnect` or `down`. The StoreState defined below will be `UP`. So, we need provide a grpc to probe  the status of TiFlash

https://github.com/pingcap/kvproto/blob/4c6f1502851ed55b3ed023d180b6b10766446630/proto/metapb.proto#L23

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What problem does this PR solve?

Issue Number: close #718

Problem Description: **TBD**

### What is changed and how does it work?

Add a grpc to detect the status of TiFlash.
No Test because
1. Mock for probe grpc is unnecessary. 
2. IT for it is a little difficult.

### Code changes

- Has exported function/method change
- Has exported variable/fields change
- Has methods of interface change
- Has persistent data change
- No code

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

### Side effects

- Possible performance regression, WHY: **TBD**
- Increased code complexity, WHY: **TBD**
- Breaking backward compatibility, WHY: **TBD**
- NO side effects

### Related changes

- Need to cherry-pick to the release branch
- Need to update the documentation
- Need to be included in the release note
- NO related changes

### CODE REVIEW

**Review by zhangyangyu — DISMISSED** [review]

File: src/main/java/org/tikv/common/region/AbstractRegionStoreClient.java [hunk 1]
```diff
@@ -66,7 +66,7 @@
           .labelNames("cluster")
           .register();
 
-  protected final RegionManager regionManager;
+  public final RegionManager regionManager;
```
Comment (pingyu) [inline src/main/java/org/tikv/common/region/AbstractRegionStoreClient.java:69]: Why change here ?

File: src/main/java/org/tikv/common/region/RegionStoreClient.java [hunk 2]
```diff
@@ -1335,6 +1338,22 @@ private void rawDeleteRangeHelper(RawDeleteRangeResponse resp) {
     }
   }
 
+  public static boolean isMppAlive(ManagedChannel channel) {
```
Comment (pingyu) [inline src/main/java/org/tikv/common/region/RegionStoreClient.java:1341]: Why add this method here ? It seems to have nothing to do with `RegionStoreClient`.

Comment (shiyuhang0) [reply src/main/java/org/tikv/common/region/RegionStoreClient.java:1341]: I also think it is not a good place to add this method. But I did not find a suitable place. Is there any suggestions?

File: src/main/java/org/tikv/common/region/AbstractRegionStoreClient.java (hunk 1 above)
Comment (shiyuhang0) [reply src/main/java/org/tikv/common/region/AbstractRegionStoreClient.java]: typo

**Comment by shiyuhang0** [comment]
closed by isMppAlive is not suitable in regionstoreclient

**Pull request closed without merging**